src/common/transformations/src/transformations/rt_info/disable_fp16_compression.cpp

```diff
@@ -5,8 +5,11 @@
 #include "transformations/rt_info/disable_fp16_compression.hpp"
 
 namespace {
-const std::string& postponed_fp16_compression_tag = "postponed_fp16_compression";
+static const std::string& get_postponed_fp16_compression_tag() {
```

Comment:
minor: we can use either static of unnamed namespace, because they do the same things

Reply:
fixed, thanks